src/content/actions/types.js

```diff
@@ -76,7 +76,8 @@ type TimelineAction =
    { type: 'CHANGE_TIMELINE_EXPANDED_THREAD', threadIndex: ThreadIndex, isExpanded: boolean };
 
 type URLEnhancerAction =
-  { type: "@@urlenhancer/urlSetupDone" };
+  { type: "@@urlenhancer/urlSetupDone" } |
+  { type: '@@urlenhancer/updateURLState', urlState: any };
```

Comment:
Yeah I think we can, its structure is well known. In a separate patch though ? :)